webroot/index.php

```diff
@@ -17,6 +17,10 @@
 
 $notices = $SQL->getNotices();
 foreach ($notices as $notice) {
+    if ($notice["expiry"] < date('Y-m-d')) {
+        $SQL->deleteNotice($notice["id"]);
```

Comment:
Doing the deletion here causes a race condition if two page loads happen at the same time and so try to delete the same items. This will cause it to throw an exception.